Change: Add a hint about following the prerequisites (#175)
greenbone/docs: src/21.4/container/index.md

## Setup and Start Script

```{note}

> Please don't forget to follow the instructions from the [Prerequisites](#prerequisites)

Reviewer: ```suggestion
Please remember to follow the instructions described in the [Prerequisites](#prerequisites)
```
